### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by jacobmolby** [stephenafamo/bob#197] [General]
Hi @stephenafamo do you know, when you are doing a new release?

The last one was made in January and there are some changes that I would really appreciate to get out.

**Comment by stephenafamo** [comment]
I'll tag a release in the next few days 😅

**Reply by jacobmolby** [reply]
@stephenafamo any updates?

**Reply by jacobmolby** [reply]
@stephenafamo If there is anything I can do to help please do tell :)

**Reply by stephenafamo** [reply]
Just wanted to squash a few more bugs. It's now only some docs updates and the release would be ready.

**Reply by stephenafamo** [reply]
Done... finally 😅

**Reply by jacobmolby** [reply]
Beautiful 😄
